src/planarGraph/boyerMyrvold.c

```diff
@@ -183,6 +183,7 @@ PGDLLEXPORT Datum _pgr_boyermyrvold(PG_FUNCTION_ARGS) {
         values[1] = Int64GetDatum(result_tuples[funcctx->call_cntr].source);
         values[2] = Int64GetDatum(result_tuples[funcctx->call_cntr].target);
         values[3] = Float8GetDatum(result_tuples[funcctx->call_cntr].cost);
+        values[4] = (result_tuples[funcctx->call_cntr].planarity);
```

Comment:
Based on:
https://www.postgresql.org/docs/12/xfunc-c.html
You will need to use:
PG_RETURN_BOOL

Also because you are not returning a table/record here wont be more than one call